apps/web/app/api/v1/client/displays/[displayId]/responded/route.ts

```diff
@@ -27,3 +27,21 @@ export async function POST(_: Request, { params }: { params: { displayId: string
     return responses.internalServerErrorResponse(error.message);
   }
 }
+export async function PUT(
+  request: Request,
+  { params }: { params: { displayId: string } }
+): Promise<NextResponse> {
+  const { displayId } = params;
+
+  if (!displayId) {
+    return responses.badRequestResponse("Missing displayId");
+  }
+  const displayUpdate = await request.json();
```

Comment:
I think we should validate this here and and return badRequest + instructions how to solve it.
See e.g. `apps/web/app/api/v1/client/responses/route.ts` for an example how to validate payload